cfme/tests/infrastructure/test_quota_tagging.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.provisioning import do_vm_provisioning
+from cfme.utils.generators import random_vm_name
+from cfme.utils.update import update
+
+pytestmark = [
+    pytest.mark.provider([RHEVMProvider], scope="module")
+]
+
+
+@pytest.fixture
+def vm_name():
+    return random_vm_name()
```

Comment:
REQUIRED: please add context. otherwise, it will be difficult to find the owner of deployed vm